ajax/libs/selectonic/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "selectonic",
+  "version": "0.6.3",
+  "description": "jQuery-plugin for making any list of items selectable by mouse and keyboard.",
+  "author": {
+    "name": "Alexey Novichkov",
+    "email": "mygrib@gmail.com"
+  },
+  "filename": "selectonic.min.js",
+  "keywords": [
+    "list",
+    "jquery",
+    "selectable",
+    "selection"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/anovi/selectonic.git"
+  },
+  "license": "MIT",
+  "homepage": "https://anovi.github.io/selectonic/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/anovi/selectonic.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "",
+        "files": [
+          "jquery.multiSelectable*"
```

Comment:
This pattern would get `jquery.multiSelectable.sublime-project`@0.1.0 which can't be added.
Let's revise to `jquery.multiSelectable*.+(js|css|map)`
Many thanks.